### DISCUSSION TITLE
Feedback on deeplink method

### DISCUSSION THREAD

**Discussion by uschouhan** [selenide/selenide#2348] [Q&A]
Found openIOSDeepLink method redirect app to one confirmation screen in which we need to click Open button manually then only desired app screen reaches . Please try handle this screen as well . 
Refer attached video . 

https://github.com/selenide/selenide/assets/118183806/4b8a601f-b757-4fb6-8526-b41a3bd2082c

**Comment by BorisOsipov** [comment]
@uschouhan "https://angelapp.page.link/blablalba" is not a deeplink. It is a regular web address used for shortening URLs.
Deeplink URLs typically follow the format "myappname://some/url/to/activity".

We are unable to handle thousands of URL shortening sites.
